src/Simulation/Simulators.Tests/App.config

```diff
@@ -3,4 +3,7 @@
   <appSettings>
     <add key="xunit.methodDisplay" value="method"/>
   </appSettings>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
```

Comment:
I'm actually not sure if it's required... Let me double check that!

Reply:
I think we just need it for the `xunit.methodDisplay` configuration, which would default to `classAndMethod` if not otherwise specified. It makes the unit test log output a little cleaner. See https://xunit.github.io/docs/configuration-files#methodDisplay.